packages/three-particle-emitter/src/index.ts

```diff
@@ -151,7 +151,8 @@ export class ParticleEmitter extends Mesh {
       fragmentShader,
       transparent: true,
       depthWrite: false,
-      fog: true,
+      // @ts-ignore
```

Comment:
Hmm yeah to fix I think you would need to create a ShaderMaterial subclass or just override the type here. This seems fine.

Reply:
Added TODO to resolve the root issue properly later so far.